engine/src/main/java/org/terasology/rendering/cameras/OpenVRStereoCamera.java

```diff
@@ -193,22 +193,22 @@ public Matrix4f getInverseViewProjectionMatrix() {
     @Deprecated
     public void loadProjectionMatrix() {
         glMatrixMode(GL_PROJECTION);
-        GL11.glLoadMatrix(getProjectionMatrix().get(BufferUtils.createFloatBuffer(16)));
+        GL11.glLoadMatrixf(getProjectionMatrix().get(BufferUtils.createFloatBuffer(16)));
         glMatrixMode(GL11.GL_MODELVIEW);
     }
 
     @Override
     @Deprecated
     public void loadModelViewMatrix() {
         glMatrixMode(GL11.GL_MODELVIEW);
-        GL11.glLoadMatrix(getViewMatrix().get(BufferUtils.createFloatBuffer(16)));
+        GL11.glLoadMatrixf(MatrixUtils.matrixToFloatBuffer(getViewMatrix()));
     }
 
     @Override
     @Deprecated
     public void loadNormalizedModelViewMatrix() {
         glMatrixMode(GL11.GL_MODELVIEW);
-        GL11.glLoadMatrix(normViewMatrix.get(BufferUtils.createFloatBuffer(16)));
+        GL11.glLoadMatrixf(MatrixUtils.matrixToFloatBuffer(normViewMatrix));
```

Comment:
```suggestion
        GL11.glLoadMatrixf(normViewMatrix.get(BufferUtils.createFloatBuffer(16)));
```
MatrixUtils is deprecated.